Gopkg.lock

```diff
@@ -256,6 +256,12 @@
   packages = ["."]
   revision = "417edcfd99a4d472c262e58f22b4bfe97580f03e"
 
+[[projects]]
+  name = "github.com/moby/moby"
```

Comment:
There's already `docker/docker` `v1.13.1` in the project, we should only have

Reply:
do we plan to move to moby?

Reply:
I don't think so, `moby` is the upstream we're more likely stay on stable docker releases

Reply:
As discussed with @hkaj we'll stick with `moby` for now and let you guys give it a try on your side